user-creation/values.yaml

```diff
@@ -2,4 +2,8 @@
 
 # projects:
 #   - name: example-project-1
-#   - name: example-project-2
\ No newline at end of file
+#     cpuLimit: 1000m
```

Comment:
maybe add some information or link how to interpret these values, e.g.
1000m means? one full CPU, a quarter of 4 CPUs or simply if abstracted, 1000 CPU milliseconds (how to interpret that?)
https://medium.com/@betz.mark/understanding-resource-limits-in-kubernetes-cpu-time-9eff74d3161b